src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -159,7 +175,7 @@ public ElementTag getToggleSessionLevelInSectionButton() {
      * @param sectionIndex the index of the section which will be checked for special privileges
      * @return             a list of checkboxes, separated into rows and columns
      */
-    private List<List<ElementTag>> createSpecialSectionsForSectionRow(List<String> sectionNames,
+    private List<List<ElementTag>> createCheckboxesForSectionLevelPermissionsOfInstructors(List<String> sectionNames,
                                                                       int panelIndex, int sectionIndex) {
```

Comment:
With the longer function name, it might now be better to shift all the parameters down to this line, so something like:

```
    private List<List<ElementTag>> createCheckboxesForSectionLevelPermissionsOfInstructors(
                                   List<String> sectionNames, int panelIndex, int sectionIndex) {
```

would work.